README.md

```diff
@@ -0,0 +1,61 @@
+<p align="center">
+  <img style="max-width: 100%;" alt="Welcome to Stream-Query-Client" src="images/projectLogo.png"/>
+</p>
+
+# Stream-Query-Client
+
+Stream-Query-Client is the easiest way to use Vectara's streaming query API. Simply provide a query configuration and a callback to instantly receive stream updates.
```

Comment:
I think we should clarify that it's the easiest way to use it in JavaScript. The reader might be looking for something to help them with a Python codebase, for example.